package.json

```diff
@@ -17,7 +17,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 6.0.0"
+    "node": ">= 8.0.0"
```

Comment:
I think we just require node 7.6.0 now given the Koa update. It's puppeteer 2.0.0 which requires 8.0.0.